code/modules/cargo/bounties/assistant.dm

```diff
@@ -0,0 +1,270 @@
+	wanted_types = list(/obj/item/clothing/glasses/sunglasses)
+
+/datum/bounty/item/assistant/gondola_hide
+	name = "Gondola Hide"
+	description = "Central Command has recently learned of strange creatures called Gondolas. If you catch one, ship its hide back to CentCom."
+	reward = 5000
+	wanted_types = list(/obj/item/stack/sheet/animalhide/gondola)
+
+/datum/bounty/item/assistant/monkey_hide
+	name = "Monkey Hide"
+	description = "One of the scientists at CentCom is interested in testing products on monkey skin. Your mission is to acquire monkey's hide and ship it."
+	reward = 1500
+	wanted_types = list(/obj/item/stack/sheet/animalhide/monkey)
+
+/datum/bounty/item/assistant/heart
+	name = "Heart"
+	description = "Commander Johnson is in critical condition after suffering a heart attack. Doctors say he needs a new heart fast. Ship one, pronto!"
+	reward = 3000
+	wanted_types = list(/obj/item/organ/heart)
+
+/datum/bounty/item/assistant/lung
+	name = "Lungs"
+	description = "A recent explosion at Central Command has left multiple staff with punctured lungs. Ship spare lungs to be rewarded."
+	reward = 3000
+	required_count = 1
+	wanted_types = list(/obj/item/organ/lungs)
+
+/datum/bounty/item/assistant/appendix
+	name = "Appendix"
+	description = "Chef Gibb of Central Command wants to prepare a meal using a very special delicacy: an appendix. If you ship one, he'll pay."
+	reward = 3000
+	wanted_types = list(/obj/item/organ/appendix)
+
+/datum/bounty/item/assistant/lizard_tail
+	name = "Lizard Tail"
+	description = "The Wizard Federation has made off with Nanotrasen's supply of lizard tails. While CentCom is dealing with the wizards, can the station spare a tail of their own?"
+	reward = 3000
+	wanted_types = list(/obj/item/organ/tail/lizard)
+
+/datum/bounty/item/assistant/shard
+	name = "Shards"
+	description = "A killer clown has been stalking CentCom, and staff have been unable to catch her because she's not wearing shoes. Please ship some shards so that a booby trap can be constructed."
+	reward = 1500
+	required_count = 15
+	wanted_types = list(/obj/item/shard)
+
+/datum/bounty/item/assistant/comfy_chair
+	name = "Comfy Chairs"
+	description = "Commander Pat is unhappy with his chair. He claims it hurts his back. Ship some alternatives out to humor him. "
+	reward = 1500
+	required_count = 5
+	wanted_types = list(/obj/structure/chair/comfy)
+
+/datum/bounty/item/assistant/revolver
+	name = "Revolver"
+	description = "Captain Johann of station 12 has challenged Captain Vic of station 11 to a duel. He's asked for help securing an appropriate revolver to use."
+	reward = 2000
+	wanted_types = list(/obj/item/gun/ballistic/revolver)
+
+/datum/bounty/item/assistant/hand_tele
+	name = "Hand Tele"
+	description = "Central Command has come up with a genius idea: Why not teleport cargo rather than ship it? Send over a hand tele, receive payment, then wait 6-8 years while they deliberate."
+	reward = 2000
+	wanted_types = list(/obj/item/hand_tele)
```

Comment:
Similiar to krav maga, irreplaceable, I'd recommend against giving incentives to sell these, especially for only 2000.